### PR TITLE
Adds Manual Approval Gate env for openshift

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
This patch adds for manual approval gate image in csv generation config.yaml

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: operatorhub/openshift/config.yaml [hunk 1]
```diff
@@ -204,6 +204,14 @@ image-substitutions:
...
         envKeys:
           - IMAGE_HUB_TEKTON_HUB_UI
+- image: registry.redhat.io/openshift-pipelines/pipelines-manual-approval-gate-rhel8@
+  replaceLocations:
+    envTargets:
+      - deploymentName: openshift-pipelines-operator
+        containerName: openshift-pipelines-operator-lifecycle
+        envKeys:
+          - IMAGE_MAG_TEKTON_TASKGROUP_CONTROLLER
+          - IMAGE_MAG_MANUAL_APPROVAL
```
Comment (jkandasa) [inline operatorhub/openshift/config.yaml:214]: @PuneetPunamiya Why do we use two type of environment keys for ManualApprovalGate image?

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml.tmpl [hunk 2]
```diff
@@ -212,6 +212,27 @@ spec:
           x-descriptors:
             - urn:alm:descriptor:com.tectonic.ui:label
       version: v1alpha1
+    - description: Represents an installation of latest version of OpenShift Manual Approval Gate
+      displayName: OpenShift Manual Approval Gate
+      kind: OpenShiftManualApprovalGate
+      name: openshiftmanualapprovalgates.operator.tekton.dev
```
Comment (jkandasa) [inline operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml.tmpl:218]: I see we use `ManualApprovalGate` as kind in CRDs.
But the kind and name looks different here. 
Is there a reason for that?

File: operatorhub/openshift/config.yaml (hunk 1 above)
Comment (PuneetPunamiya) [reply operatorhub/openshift/config.yaml:214]: We have two deployments in manual approval, one for controller and one for webhook

Comment (jkandasa) [reply operatorhub/openshift/config.yaml:214]: thanks @PuneetPunamiya 
however the container name looks bit odd :)
* `tekton-taskgroup-controller` => ManualApprovalGate Controller
* `manual-approval` => ManualApprovalGate Webhook

**Review by jkandasa — APPROVED** [review]

**Pull request merged**